editor/groups_editor.cpp

```diff
@@ -217,19 +217,14 @@ void GroupDialog::_add_group_text_changed(const String &p_new_text) {
 }
 
 void GroupDialog::_group_renamed() {
-	TreeItem *renamed_group = groups->get_edited();
+	TreeItem *renamed_group = groups->get_selected();
```

Comment:
This was changed to prevent bug:
![group_renaming_bug](https://user-images.githubusercontent.com/3036176/177032700-6985e66e-f9ec-49ed-9caa-6f96cf867822.gif)